test/shim/test_sequencer.py

```diff
@@ -130,6 +130,13 @@ def test_shim_sequencer_lsq(self, nii_fieldmap, nii_anat, nii_mask, sph_coil, sp
         currents = sequencer_test.shim()
         sequencer_test.eval(currents)
         assert_results(nii_fieldmap, nii_anat, nii_mask, [sph_coil], currents, slices)
+    def test_shim_sequencer_lsq(self, nii_fieldmap, nii_anat, nii_mask, sph_coil, sph_coil2):
```

Comment:
- Make sure all test names make sense, I sometimes wrote lsq because it was the default, now that quad prog is the default, they might not make sense anymore. Also make sure that it does not say lsq when it is qp

Reply:
Yes,  it should be good now !